components/pkg-export-container/src/cli.rs

```diff
@@ -19,22 +19,20 @@ pub fn cli() -> Command {
     let about = "Creates a container image from a set of Habitat packages (and optionally pushes \
                  to a remote repository)";
 
-    let cmd =
-        Command::new(name).max_term_width(80)
-                          .about(about)
-                          .version(VERSION)
-                          .author("\nAuthors: The Habitat Maintainers <humans@habitat.sh>")
-                          .help_template("{name} {version} {author-section} {about-section} \
-                                          \n{usage-heading} {usage}\n\n{all-args}")
-                          .arg(Arg::new("IMAGE_NAME").long("image-name")
-                                                     .short('i')
-                                                     .value_name("IMAGE_NAME")
-                                                     .help("Image name template: supports: \
-                                                            {{pkg_origin}}/{{pkg_name}} \
-                                                            {{pkg_origin}}, {{pkg_name}}, \
-                                                            {{pkg_version}}, {{pkg_release}}, \
-                                                            {{channel}})")
-                                                     .default_value("{{pkg_origin/pkg_name}}"));
+    let cmd = Command::new(name).max_term_width(120)
+                                .about(about)
+                                .version(VERSION)
+                                .author("\nAuthors: The Habitat Maintainers <humans@habitat.sh>")
+                                .help_template("{name} {version} {author-section} {about-section} \
+                                                \n{usage-heading} {usage}\n\n{all-args}")
+                                .arg(Arg::new("IMAGE_NAME").long("image-name")
+                                                           .short('i')
+                                                           .value_name("IMAGE_NAME")
+                                                           .help("Image name template: supports: \
+                                                                  {{pkg_origin}}/{{pkg_name}} \
+                                                                  (default), {{pkg_origin}}, \
```

Comment:
I think you can remove `{{pkg_origin}}/{{pkg_name}}` and `(default)`. It's just {{pkg_origin}}, {{pkg_name}}, {{pkg_version}}, {{pkg_release}}, and {{channel}} that we can substitute.

Reply:
Updated the help text